releasenotes/notes/asm-blocking-fixes-fce3bb6a81a2d186.yaml

```diff
@@ -0,0 +1,9 @@
+---
+fixes:
+  - |
+    Solve some corner cases where a Flask blocking request would fail because 
```

Comment:
Do we want to pre-pend each section with `ASM:` so that users know what part of the library we're talking about?

Reply:
You're right.